DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/io_streaming/src/FakeIOPipes.hpp

```diff
@@ -3,6 +3,7 @@
 

```

Comment:
I assume this file is now identical to the copy in the mvdr design?  (If this is not true - why not?)

Reply:
Almost. The one here has a comment explaining the `detail` namespace. Otherwise they are identical.

Reply:
I didn't like my answer. They are now the same.